src/PageContainer.js

```diff
@@ -19,6 +20,13 @@ class PageContainer extends Component {
 
         this.state = {
             content: props.content,
+            showPopUp: false,
+            hover: false,
+            // TODO: how to handle multiple notifications
```

Comment:
This seems like a pretty important question - we could totally have them have multiple notifications within the first week of the pilot? I'd ask Madison and try to figure this out. Also are we handling the little red number on the bell icon that indicates how many notifications they have?